### ISSUE TITLE
print layer

### ISSUE THREAD

**Issue by Alexis-Martin** [saucisson/lua-layeredata#7]
When we print a layer it could be great to choose if we want print __meta__ or not and choose if we want the proxy or not

    local Serpent = require "serpent"
    local Proxy   = require "layeredata"
    local layer   = Proxy.new { 
      name = "print example",
    }
    local _       = Proxy.placeholder

    layer.__meta__ = {
      print_type = {
        __meta__ = {
          A_type = {},
          B_type = {
            content_type    = _.__meta__.print_type.__meta__.A_type,
            value_container = _.__meta__.print_type.A,
          },
        },
        
        A = {
          __meta__ = {
            content_type = _.__meta__.print_type.__meta__.A_type,
          },
          
          a = {
            __refines__ = {
              _.__meta__.print_type.__meta__.A_type,
            },
          },
        },
        
        B = {
          __meta__ = {
            content_type = _.__meta__.print_type.__meta__.B_type,
          },
          b = _.__meta__.print_type.A.a,
        },
      },
    }
    local function dump (x)
      return Serpent.dump (x, {
        indent   = "  ",
        comment  = false,
        sortkeys = true,
        compact  = false,
      })
    end
    print(dump(Proxy.flatten(layer)))

In this example the element b is : 

    b = {
       [1] = "__meta__",
       [2] = "print_type",
       [3] = "A",
       [4] = "a",
       __layer = "print example",
       __proxy = true
   }

and I would have : 

    b = __meta__.print_type.A.a

for example

**Comment by saucisson** [comment]
In the "values" branch, you can now test:

````lua
    Layer.dump (proxy) -- user-friendly
    Layer.dump (proxy, true) -- computer-friendly
````

**Comment by saucisson** [comment]
Fixed in 9dd795a8f935b5cb32ecec1bbb768e14ca3bfeba
